LongWarOfTheChosen/Config/XComGameData_WeaponData.ini

```diff
@@ -370,6 +370,13 @@ Sectopod_idealRange=12
 AndromedonRobot_idealRange=1
 PsiZombie_idealRange=1
 
+-GREMLINMK2_ABILITYDAMAGE=(Damage=3, Spread = 0, PlusOne = 50, Crit = 0, Pierce = 1000, Tag = "CombatProtocol_Robotic", DamageType="Electrical")
+-GREMLINMK3_ABILITYDAMAGE=(Damage=4, Spread = 0, PlusOne = 50, Crit = 0, Pierce = 1000, Tag = "CombatProtocol_Robotic", DamageType="Electrical")
+
++GREMLINMK2_ABILITYDAMAGE=(Damage=4, Spread = 0, PlusOne = 50, Crit = 0, Pierce = 1000, Tag = "CombatProtocol_Robotic", DamageType="Electrical")
++GREMLINMK3_ABILITYDAMAGE=(Damage=6, Spread = 0, PlusOne = 50, Crit = 0, Pierce = 1000, Tag = "CombatProtocol_Robotic", DamageType="Electrical")
```

Comment:
Similar to the Bluescreen changes, I'm not sold on buffing Combat Protocol. It's just one more balance change that doesn't feel really necessary, even if it's not a bad idea per se.